OSPSuite.Presentation.Importer/Core/DataFormat/DataFormat_TMetaData_C.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using OSPSuite.Core.Importer;
+
+namespace OSPSuite.Presentation.Importer.Core.DataFormat
+{
+   public class DataFormat_TMetaData_C : AbstractColumnsDataFormat
+   {
+      public override string Name { get; } = "015_TMetaData_C(E)";
```

Comment:
Same here. Move all of that to constant files (Name and Description)

Reply:
Done